aim/engine/utils.py

```diff
@@ -1,311 +0,0 @@
-        obj_name = inst.__name__
-    if hasattr(inst, '__module__'):
-        obj_module = inst.__module__
-    if hasattr(inst, '__class__'):
-        if hasattr(inst.__class__, '__name__'):
-            obj_cls_name = inst.__class__.__name__
-        if hasattr(inst.__class__, '__module__'):
-            obj_cls_module = inst.__class__.__module__
-
-    obj_full = '{}.{}'.format(obj_name, obj_module)
-    obj_cls_full = '{}.{}'.format(obj_cls_name, obj_cls_module)
-
-    return obj_full, obj_cls_full
-
-
-def get_inst_base_types(inst):
-    """
-    Get instance and it's base classes types
-    """
-    bases_types = []
-    for b in inst.__class__.__bases__:
-        b_type, b_cls_type = get_inst_type_str(b)
-        bases_types.append(b_type)
-        bases_types.append(b_cls_type)
-    return bases_types
-
-
-def inst_has_typename(inst, types):
-    """
-    Return `True` if the instance is created from class
-    which has base that matches passed `types`
-    """
-    inst_type, inst_cls_type = get_inst_type_str(inst)
-    inst_types = [inst_type, inst_cls_type] + get_inst_base_types(inst)
-
-    for i in inst_types:
-        found = True
-        for t in types:
-            if i.find(t) == -1:
-                found = False
-                break
-        if found:
-            return True
-
-    return False
-
-
-def is_keras_model(inst):
-    """
-    Check whether `inst` is instance of keras model
-    """
-    return inst_has_typename(inst, ['keras', 'Model'])
-
-
-def is_pytorch_module(inst):
-    """
-    Check whether `inst` is instance of pytorch module
-    """
-    return inst_has_typename(inst, ['torch', 'Module'])
-
-
-def is_tensorflow_session(inst):
-    """
-    Check whether `inst` is instance of tensorflow session
-    """
-    return inst_has_typename(inst, ['tensorflow', 'session'])
-
-
-def is_tensorflow_estimator(inst):
-    """
-    Check whether `inst` is instance of tensorflow estimator
-    """
-    return inst_has_typename(inst, ['tensorflow', 'estimator'])
-
-
-def is_pytorch_optim(inst):
-    """
-    Check whether `inst` is instance of pytorch optimizer
-    """
-    return inst_has_typename(inst, ['torch', 'optim'])
-
-
-def is_pytorch_tensor(inst):
-    """
-    Check whether `inst` is instance of pytorch tensor
-    """
-    return inst_has_typename(inst, ['torch', 'Tensor'])
-
-
-def is_numpy_array(inst):
-    """
-    Check whether `inst` is instance of numpy array
-    """
-    return inst_has_typename(inst, ['numpy', 'ndarray'])
-
-
-def is_numpy_number(inst):
-    """
-    Check whether `inst` is numpy number
-    """
-    return inst_has_typename(inst, ['numpy'])
-
-
-def is_number(value):
-    """
-    Checks if the given value is a number
-    """
-    if isinstance(value, (int, float)):
-        return True
-
-    if is_numpy_number(value):
-        return True
-
-    if is_pytorch_tensor(value):
-        return True
-
-    return False
-
-
-def convert_to_py_number(value):
```

Comment:
if I am not wrong, this function is used in `sdk/adapters` package.

Reply:
will double-check this one